WhatsApp/app/src/main/java/com/example/whatsapp/ui/fragments/settings/chats/SettingsChatsFragment.kt

```diff
@@ -0,0 +1,71 @@
+package com.example.whatsapp.ui.fragments.settings.chats
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import androidx.fragment.app.Fragment
+import androidx.navigation.NavController
+import com.example.whatsapp.R
+import com.example.whatsapp.base.BaseFragment
+import com.example.whatsapp.databinding.FragmentNotificationsBinding
+import com.example.whatsapp.databinding.FragmentSettingsChatBinding
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class SettingsChatsFragment : BaseFragment<FragmentSettingsChatBinding>() {
+
+
+    override fun inflateBinding(layoutInflater: LayoutInflater): FragmentSettingsChatBinding {
+        return FragmentSettingsChatBinding.inflate(layoutInflater)
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        setHasOptionsMenu(true)
```

Comment:
We wont need `onCreate` in this instance `setHasOptionsMenu` can be moved to `initViews` also